README.md

```diff
@@ -75,19 +75,21 @@ syntax.
 ## Examples
 
 ```rust
-use async_std::task;
+async fn say_hello() {
+    println!("Hello, world!");
+}
 
-fn main() {
-    task::block_on(async {
-        println!("Hello, world!");
-    })
+#[async_std::main]
```

Comment:
I thought it would be nice to show that the README needs to enable the 'attributes' feature as well as the documentation.